.github/workflows/platformDeploy.yml

```diff
@@ -243,9 +243,7 @@ jobs:
 
       - name: Set iOS version in ENV
         if: ${{ fromJSON(env.SHOULD_DEPLOY_PRODUCTION) }}
-        run: |
-          echo "IOS_VERSION=$(echo '${{ github.event.release.tag_name }}' | tr '-' '.')" >> $GITHUB_ENV
-          echo "iOS version is: ${{ env.IOS_VERSION }}"
+        run: echo "IOS_VERSION=$(echo '${{ github.event.release.tag_name }}' | tr '-' '.')" >> $GITHUB_ENV
```

Comment:
Can we add the echo back in for debugging purposes, or is there something functionally different here?

Reply:
It doesn't work using this one line syntax, I think the entire line needs to be executed, then we could run this. I verified this here: https://github.com/Andrew-Test-Org/Public-Test-Repo/actions/runs/1195174314/workflow#L13-L19

<img width="1032" alt="Screen Shot 2021-09-02 at 4 43 29 PM" src="https://user-images.githubusercontent.com/2838819/131925522-c7831158-d036-485f-a737-231976d0f7d0.png">

I personally was using the ENV variables to debug this just as easily:

<img width="1029" alt="Screen Shot 2021-09-02 at 4 43 00 PM" src="https://user-images.githubusercontent.com/2838819/131925565-0f95ef17-42e5-4f0b-802d-e7243737e2eb.png">

Happy to add it into it's own step if you think it's valuable, I just thought it was duplicated.

Reply:
Nice, it's obviously visible there.